### DISCUSSION TITLE
Not able to add a user to an organization created by me

### DISCUSSION THREAD

**Discussion by Kartikey-star** [artifacthub/hub#1987] [Q&A] [answered]
I have created an organization in artifact hub and i am not able to add one particular user to it. I am searching with the user's github username and still get user not found error. It works with other user id's  though. I will be grateful if you can provide some insights on that.

**Comment by tegioz** [comment, marked as answer]
Hi @Kartikey-star 👋 

Before adding a user, they need to be registered in Artifact Hub. The username to provide when adding a user to an organization should be the one used in Artifact Hub, not in GitHub (they can match though).

Hope this helps 🙂

**Reply by Kartikey-star** [reply]
Well the user is registered in artifacthub too and i have searched with that username  as well .So what are we possibly missing?

**Reply by tegioz** [reply]
Let me look into it. Could you please share the Artifact Hub username of the user you are trying to invite to the organization?

**Reply by Kartikey-star** [reply]
Thanks.The artifacthub username is jchui .

**Reply by tegioz** [reply]
You should be able to add the user now. I think there is an issue that may be affecting some more users, working on it now.

Thanks for raising this!

**Reply by Kartikey-star** [reply]
Yeah i am able to add the user now. Thank you.
